.github/workflows/wheels-manylinux-test.yml

```diff
@@ -85,8 +85,10 @@ jobs:
             ],
             "nightly": [
               { "arch": "amd64", "python": "3.8", "ctk": "11.8.0", "image": "ubuntu18.04", "test-type": "unit", "test-command": "${{ inputs.test-unittest }}", "gpu": "v100", "driver": "525" },
+              { "arch": "amd64", "python": "3.9", "ctk": "11.8.0", "image": "ubuntu18.04", "test-type": "unit", "test-command": "${{ inputs.test-unittest }}", "gpu": "v100", "driver": "525" },
               { "arch": "amd64", "python": "3.10", "ctk": "11.8.0", "image": "ubuntu18.04", "test-type": "unit", "test-command": "${{ inputs.test-unittest }}", "gpu": "v100", "driver": "525" },
               { "arch": "arm64", "python": "3.8", "ctk": "11.8.0", "image": "ubuntu20.04", "test-type": "unit", "test-command": "${{ inputs.test-unittest }}", "gpu": "a100", "driver": "525" },
+              { "arch": "arm64", "python": "3.9", "ctk": "11.8.0", "image": "ubuntu20.04", "test-type": "unit", "test-command": "${{ inputs.test-unittest }}", "gpu": "a100", "driver": "525" },
```

Comment:
```suggestion
```

can we drop the `3.9` tests for `arm`?

`arm` resources are limited (only 8 runners).

adding this entry would mean each PR alone takes up 3 `arm` runners just for `wheels` (not including `conda`).